apps/token-manager/app/src/screens/Holders.js

```diff
@@ -62,11 +63,15 @@ class Holders extends React.Component {
                           groupmode={groupMode}
                         />
                         {!groupMode && (
-                          <StyledTableHeader title="Balance" align="right" />
+                          <StyledTableHeader
+                            title={<div css="margin-left: -20px">Balance</div>}
```

Comment:
This seems quite weird; did we modify the `TableHeader` to include internal padding? Could this be moved to the `StyledTableHeader` definition?

Reply:
I know 😩 it’s really specific to this app, and the only way I found to gain enough space to remove the horizontal scrollbar without compromising the layout too much, by letting the “Balance” header overlapping 20px on the “Holder” header:

<img width="330" alt="image" src="https://user-images.githubusercontent.com/36158/54627585-09a54a80-4a74-11e9-8668-b6aed0a83f82.png">

Even at `320px` there is plenty of space in the “Holder” header, so it shouldn’t be a problem… but I should add a comment at the very least.

Also I don’t think we should add a prop for now, it’s more something that should be avoided when the screen gets designed.

Reply:
Maybe we could rename it to something else, like `BLN`. I think it's pretty clear what this column is.

Reply:
I agree it should stay pretty clear, but still not be as clear as having it entirely, and we have enough space to do it? Do you see a problem with using a negative margin here?

Reply:
Looks a bit weird with the "you" sticking half way into the balance header 😄.

Less of a problem if you're not the first token holder though.

Reply:
Oh I see, I thought it was about the code! I’d really like to avoid abbreviations if we have other options. What would you think about having it left aligned & fusioned with the next cell on small viewports?

<img width="330" alt="image" src="https://user-images.githubusercontent.com/36158/54638556-9f989f80-4a8b-11e9-9f7a-6a3aec5ff7a1.png">

Reply:
Maybe this is better, especially since the `HOLDER` header is left-aligned (and technically the action buttons _modify_ the balance).

Reply:
Done!